src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -290,6 +290,37 @@ public Job call() throws SaltStackException {
     }
 
     /**
+     * Query for result of supplied job.
+     * 
```

Comment:
Nitpick: there's a trailing space on this line (and others below). Please get rid of all trailing spaces.